app/models/concerns/request_item.rb

```diff
@@ -25,7 +39,44 @@ def duke_owned?
     record_owner == 'duke'
   end
 
+  def trln_partner_owned?
+    record_owner.in?(%w[unc ncsu nccu])
+  end
+
   def physical_item?
     fetch(TrlnArgon::Fields::ACCESS_TYPE, []).include?('At the Library')
   end
+
+  # These values are consistent, no matter the item or the home library's
+  # ILLiad instance
+  def illiad_request_params_fixed
+    {
+      'Value': 'GenericRequestTRLNLoan',
+      'genre': 'TRLNbook',
+      'RequestType': 'null',
+      'form': '20',
+      'Action': '10',
+      'IlliadForm': 'Logon',
+      'CitedPages': 'TRLN'
+    }
+  end
+
+  def illiad_request_params_variable
+    {
+      'Location': item_url_absolute,
+      'LoanTitle': fetch(TrlnArgon::Fields::TITLE_MAIN),
```

Comment:
@seanaery do you want to provide a default value if the fetch fails -- `fetch(TrlnArgon::Fields::TITLE_MAIN, '')`. It _shoudn't_ fail because title main is a required field, but it's probably good practice just in case. Otherwise, the default behavior with fetch is to return a `KeyError (key not found: title_main).`

Reply:
@corylown Great idea, yes, I'll make that change.